setup.py

```diff
@@ -19,52 +19,54 @@
 package_name = 'onnxruntime'
 wheel_name_suffix = None
 
+def parse_arg_remove_boolean(argv, arg_name):
+    arg_value = False
+    if arg_name in sys.argv:
+        arg_value = True
+        argv.remove(arg_name)
+
+    return arg_value
+
+def parse_arg_remove_string(argv, arg_name_equal):
+    arg_value = None
+    for arg in sys.argv[1:]:
+        if arg.startswith(arg_name_equal):
+            arg_value = arg[len(arg_name_equal):]
+            sys.argv.remove(arg)
+            break
+
+    return arg_value
+
 # Any combination of the following arguments can be applied
-if '--use_featurizers' in sys.argv:
-    featurizers_build = True
-    sys.argv.remove('--use_featurizers')
+featurizers_build = parse_arg_remove_boolean(sys.argv, '--use_featurizers')
 
-if '--nightly_build' in sys.argv:
+if parse_arg_remove_boolean(sys.argv, '--nightly_build'):
     package_name = 'ort-nightly'
```

Comment:
Are we going to keep "ort-nightly" as opposed to "onnxruntime-nightly"?

Reply:
The plan is to follow https://www.python.org/dev/peps/pep-0440/#file-name-convention and also stay close to pytorch versioning convention. 
This PR only affects naming and versioning for onnxruntime training packages. I just revisited the PEP440 spec made another commit so that:
    # stable:
    # onnxruntime_training-1.7.0+cu11.1-cp36-cp36m-linux_x86_64.whl
    # nightly:
    # onnxruntime_training-1.7.0.dev20210401+cu11.1-cp36-cp36m-linux_x86_64.whl

there is no 'nightly' anymore with training package. Developmental releases (PEP440) is used instead.

Reply:
I like this solution